DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,223 @@
+        private IWebElement MonthSeptember => driver.FindElement(By.XPath("//option[@value='September']"));
+        private IWebElement MonthOctober => driver.FindElement(By.XPath("//option[@value='October']"));
+        private IWebElement MonthNovember => driver.FindElement(By.XPath("//option[@value='November']"));
+        private IWebElement MonthDecember => driver.FindElement(By.XPath("//option[@value='December']"));
+        private IWebElement YearDropDown => driver.FindElement(By.XPath("//select[@id='year']"));
+        private IWebElement Year2024 => driver.FindElement(By.XPath("//option[@value='2024']"));
+        private IWebElement Year2028 => driver.FindElement(By.XPath("//option[@value='2028']"));
+        private IWebElement CalculateBtn => driver.FindElement(By.XPath("//button[@id='calculateBtn']"));
+        private IWebElement IncomeFld => driver.FindElement(By.XPath("//th[text()='Income: *']/..//input"));
+        private IWebElement InterestEarnedFld => driver.FindElement(By.XPath("//th[text()='Intereset earned: *']/..//input"));
+        private IWebElement EndDateFld => driver.FindElement(By.XPath("//th[text()='End date: *']/..//input"));
+
+        public void MandatoryTextFields(string depositAmount, string interestRate, string investmentTerm)
+        {
+            DepositAmountFld.SendKeys(depositAmount);
+            InterestRateFld.SendKeys(interestRate);
+            InvestmentTermFld.SendKeys(investmentTerm);
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => CalculateBtn.GetAttribute("disable") != string.Empty);
+        }
+
+        public void ValidCalculation(double depositAmount, double interestRate, double investmentTerm)
+        {
+            DepositAmountFld.SendKeys(Convert.ToString(depositAmount));
+            InterestRateFld.SendKeys(Convert.ToString(interestRate));
+            InvestmentTermFld.SendKeys(Convert.ToString(investmentTerm));
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => CalculateBtn.GetAttribute("disable") != string.Empty);
+        }
+
+        public void ClickOnFinancialYear365RadioBtn()
+        {
+            FinancialYear365Days.Click();
+        }
+
+        public void ClickOnFinancialYear360RadioBtn()
+        {
+            FinancialYear360Days.Click();
+        }
+
+        public void ClickOnCalculateBtn()
+        {
+            CalculateBtn.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => InterestEarnedFld.GetAttribute("value") != null);
+        }
+
+        public bool GetCalculateBtnCurrentStatus()
+        {
+            if (CalculateBtn.GetDomAttribute("disabled") != null)
+            {
+                return true;
+            }
+            return false;
+        }
+
+        public string GetInterestEarnedFldValue()
+        {
+            return InterestEarnedFld.GetAttribute("value").Replace('.', ',');
+        }
```

Comment:
You can simplify it to property:
```suggestion
        public string InterestEarned => InterestEarnedFld.GetAttribute("value");
```
Also, I don't think you should change dots with commas.